pyproject.toml

```diff
@@ -23,7 +23,8 @@ chardet = "^4.0.0"
 click = "^8.1.3"
 
 [tool.poetry.dev-dependencies]
-csvcubed-devtools = "0.1.9"
+csvcubed-models = "0.1.9"
```

Comment:
csvcubed-pmd should not have any depedencies on models, it should access everything directly from csvcubed. Can you confirm this is indeed necessary?

Reply:
I did some searching to see why models was needed here (There were pyright errors when I tried just removing it). It turns out pmd was using csvcubed-models 0.1.6 from csvcubed which meant it was not compatible with the recent changes made for dcat.distribution. I think making it use the latest version of csvcubed should fix that and remove any need for models to be in the dependencies, trying it now